canonical-web-and-design/maas-ui
# feat(ui): edit aliases and vlans form

## Done

- Add a form to edit aliases and vlans on the network tab.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the machine details network tab for a machine with editable network.
- Create a VLAN and an alias if it doesn't already have them.
- Using the action menu click edit vlan/alias.
- Change the details and save.
- The row should update for the vlan/alias.

## Fixes

Fixes: #2225.
